components/NavMenu.js

```diff
@@ -50,21 +64,30 @@ const NavWrapper = styled.nav`
 const NavButton = styled.div`
```

Comment:
Maybe you could use a Button element for the NavButton to keep it semantic. :)

Reply:
nice one, just got that implemented :)